phpBB/includes/functions_upload.php

```diff
@@ -899,6 +909,18 @@ function remote_upload($upload_url, \phpbb\mimetype\guesser $mimetype_guesser =
 						$file = new fileerror($user->lang[$this->error_prefix . 'URL_NOT_FOUND']);
 						return $file;
 					}
+					else if (stripos($line, 'location: ') !== false)
+					{
+						//there is a redirect, follow up to 5
+						if ($redirect_count > 5)
```

Comment:
Exactly, so this condition is true for $redirect_count = 6, that mean after the 6th redirect, or we want to stop before the 6th.